components/layout/OpenAuthoringSiteForm.tsx

```diff
@@ -158,10 +165,10 @@ const OpenAuthoringSiteForm = ({
       const errorUIDescriptor: OpenAuthoringContextualErrorUI = await interpretError(
         err
       )
-      if (errorUIDescriptor.asModal) {
+      if (errorUIDescriptor && errorUIDescriptor.asModal) {
         setInterpretedError(errorUIDescriptor)
       } else {
-        cms.alerts.error(errorUIDescriptor.message)
+        cms.alerts.error(errorUIDescriptor?.message || 'Could not interpret.')
```

Comment:
This should be a more generic error since it's user facing, e.g "Oops, Something went wrong"